### ISSUE TITLE
Good work

### ISSUE THREAD

**Issue by chrisrusso** [mariuszciaston/Grab-All-Magnets#1]
Quick, simple, clean, nice JS. Good work. Thank you.

**Comment by mariuszciaston** [comment]
> .

Thanks, nice to see that someone found it useful. 🧲

**Comment by chrisrusso** [comment]
Yes, it's a nice approach... it would be nice to extend this technology a little bit more... beyond a browser extension. I'll try to be brief, I really like torrents, P2P technology, decentralization, blockchain and all that is related. Here you can see a little bit more about myself, this is 10 years old: https://krebsonsecurity.com/2010/07/pirate-bay-hack-exposes-user-booty/ 

Now, going back to the topic... you made a regular expression to capture magnet links. Simple and effective... Now, for example do you have experience with crawlers? I could suscribe to the RSS feed from TPB, for example, and start downloading automatically all that comes up, however, I can't create a custom RSS feed from a keyword or search.

I do have a box, with about 72 Terabytes to make a proof of concept, which I suspect, won't be enough. Are you interested in developing something bigger? Looking forward to hear from you. GNU, opensource... we can only provide architecture instructions and some financial support.

**Comment by mariuszciaston** [comment]
Ok, that's interesting. I did some tests with crawling images and texts from one Polish website just for curiosity. I like the automated nature of it. I am open to learn new things and do something exciting! Currently i am doing The Odin Project and my most complex project as of today is todo list app: https://github.com/mariuszciaston/minimaLIST

I would like to hear more from you.

**Comment by chrisrusso** [comment]
Your "Odin Project" have nice retro sounds... I like what you are doing... even tho it's simple, I think it's cool. Keep up the good work. Feel free the message me if you need me.
